res/eg/1d/jupyter/beam/beam.py

```
@@ -0,0 +1,116 @@
+            write_timestamps=timestamps,
+        )
+
+    return sim
+
+
+def main():
+    import sys
+    from pyphare.simulator.simulator import Simulator
+
+    if len(sys.argv) != 5:
+        raise RunTimerError('This code needs 4 paramaters, "run_name", U0, Te, Ti')
```

Ruff (0.15.20)

[error] Undefined name `RunTimerError`

(F821)